### ISSUE TITLE
Calcite-radio-button-group: values do not update when wrapped in Label

### ISSUE THREAD

**Issue by evanmosby** [Esri/calcite-components#601]
Found at: Beta 26

When a `radio-button-group` is wrapped in a `label`, the checked value seems to iterate back to the first child `radio-button` 

Test repo here:
https://github.com/evanmosby/calcite-radio-button-group-test

**Comment by macandcheese** [comment]
cc @eriklharper

**Comment by eriklharper** [comment]
@evanmosby Can you add steps to reproduce for this bug?

**Comment by evanmosby** [comment]
When wrapping a radio button group in a label - I see this in the latest Beta

```
    <calcite-label>
      My Label
      <calcite-radio-button-group name="withLabel">
        <calcite-radio-button value="stencil" checked>Stencil</calcite-radio-button>
        <calcite-radio-button value="react">React</calcite-radio-button>
        <calcite-radio-button value="ember">Ember</calcite-radio-button>
        <calcite-radio-button value="angular">Angular</calcite-radio-button>
      </calcite-radio-button-group>
    </calcite-label>
```

![2020-09-02_14-00-58](https://user-images.githubusercontent.com/28676725/92036255-cd77af80-ed24-11ea-8c77-b548e8cf6767.gif)

**Comment by eriklharper** [comment]
Gotcha, looks like this issue is a dupe of #898 fixed by #915
